src/main/python/pybuilder_aws_plugin/__init__.py

```diff
@@ -4,22 +4,35 @@
 
 import sys
 
-from pybuilder.core import init
+from pybuilder.core import init, task, depends
 
-from .lambda_tasks import upload_zip_to_s3, package_lambda_code
+from .lambda_tasks import upload_zip_to_s3, package_lambda_code, lambda_release
 
 if sys.version_info[0:2] >= (2, 7):
-    from .cfn_tasks import upload_cfn_to_s3
+    # cfn-sphere needs Python 2.7, skip cfn handling for older Python versions
+    from .cfn_tasks import upload_cfn_to_s3, cfn_release
 
+    @task(description='Release on S3 by copying everything to latest')
+    @depends('lambda_release',
+             'cfn_release')
+    def release_custom_resource():
```

Comment:
I disagree. This project suffers a little bit from [premature generalization](http://c2.com/cgi/wiki?PrematureGeneralizationIsEvil). The `upload_zip_to_s3` task already did more than it said. And now we learned that this was actually wrong.

I would really prefer to name the things for the purpose at hand and see later which generalizations evolve. For CFN custom resources we usually have Python code and a CFN template. So let's just call the tasks exactly to match this use case. When we will have more uses cases that call for an `aws_release` task then we can still build that.

That is also the reason why I explicitly listed our specific use cases at the top of the documentation.